backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt

```diff
@@ -163,6 +166,25 @@ class CardMutationService {
         return activationCodes
     }
 
+    @GraphQLDescription("Creates a new digital koblenz card and returns it")
```

Comment:
i think this endpoint shouldn't be part of this pr.
This task is about hashing the user data.

Some explanations from my side:
I think this function will be used later if the user sends data from the self service portal.
Then we should receive a proper user data object which we again hash and check if the hash is in the database.
Then we create from the userData the cardData and send them back to the frontend